### PR TITLE
Migrated administrator application from FBV to CBV

### PR DESCRIPTION
**Pull request by amruthasangeeth**
Migrated the report function in administrator app to CBV using two views inside one. GenerateReportView in administrator app used ShowFormView and ShowReportListView to display the form and generate the report respectively.

### CODE REVIEW

**Comment by tapasweni-pathak** [comment]
@amruthasangeeth Number of tests failing in this pull requests are also 80.

78 out of 80 are saying unable to local element as @smarshy has already pointed out in #341. 

@amruthasangeeth Before opening a pull request run all the test cases. 
Let us know if you are not able to fix any.

**Comment by tapasweni-pathak** [comment]
@amruthasangeeth Did you get the time to fix issues in this pull request?

**Comment by amruthasangeeth** [comment]
@smarshy All test cases are passed.

File: vms/administrator/views.py [hunk 1]
```diff
@@ -1,62 +1,66 @@
...
-                )
-            total_hours = calculate_total_report_hours(report_list)
-            return render(request, 'administrator/report.html', {'form': form, 'report_list': report_list, 'total_hours': total_hours, 'notification': True, 'organization_list': organization_list, 'selected_organization': organization, 'event_list': event_list, 'selected_event': event_name, 'job_list': job_list})
-        else:
-            return render(request, 'administrator/report.html', {'form': form, 'notification': False, 'organization_list': organization_list, 'event_list': event_list, 'job_list': job_list})
-    else:
-        form = ReportForm()
-        return render(request, 'administrator/report.html', {'form': form, 'notification': False, 'organization_list': organization_list, 'event_list': event_list, 'job_list': job_list})
+    event_list = get_events_ordered_by_name()
+    job_list = get_jobs_ordered_by_title()
```
Comment (tapasweni-pathak) [inline vms/administrator/views.py]: @amruthasangeeth Please run pep8 on this file.

File: vms/administrator/views.py [hunk 2]
```diff
@@ -1,62 +1,66 @@
...
-        admin = user.administrator
-    except ObjectDoesNotExist:
-        pass
-    if not admin:
-        return render(request, 'vms/no_admin_rights.html')
+#Displays the form
+class ShowFormView(LoginRequiredMixin, FormView):
+    model = Administrator
+    form_class = ReportForm
+    template_name = "administrator/report.html"
```
Comment (tapasweni-pathak) [inline vms/administrator/views.py]: @amruthasangeeth Where are we handling this part in the new code?

Comment (amruthasangeeth) [inline vms/administrator/views.py]: @tapasweni-pathak  Ya that is not done. I will do that soon

Comment (tapasweni-pathak) [inline vms/administrator/views.py]: @amruthasangeeth Does not handling the not admin case in this pull request helps you in something? Any reason you have not handled this case with these changes?

**Pull request merged**